app/src/main/java/com/dicoding/picodiploma/sejiwaproject/db/LeagueData.kt

```diff
@@ -0,0 +1,59 @@
+package com.dicoding.picodiploma.sejiwaproject.db
+
+import com.dicoding.picodiploma.sejiwaproject.R
+import com.dicoding.picodiploma.sejiwaproject.model.League
+
+object LeagueData {
```

Comment:
Nice!
Consider to move all array to resource `strings.xml`.

Reply:
resolved, thank's